src/contexts/DataContext/DataContext.jsx

```diff
@@ -21,6 +28,9 @@ const DataProvider = ({ children }) => {
     leafy: false,
     marrow: false,
     root: false,
+    alltime: false,
```

Comment:
You can put this properties leafy,marrow and root into a separate object

Reply:
Ok, I will look into this.